### ISSUE TITLE
failing containerized tests

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproteomics/tools-galaxyp#426]
67 test(s) out of 249 fail containerized tests: https://github.com/bernt-matthias/tools-galaxyp/suites/556074621/artifacts/3554156

Please edit this message if you adopt one of the problems or solved it.

- [ ] dbbuilder `zcat: invalid magic`
- [ ] gffcompare_to_bed `NameError: name 'unicode' is not defined
- [ ] hardklor `python: command not found`
- [ ] validate_fasta_database output diff
- [ ] fasta_merge_files_and_filter_unique_sequences  output diff
- [ ] DecoyDatabase symbol `lookup error`
- [ ] MetaProSIP `MetaProSIP: command not found`
- [ ] MzMLSplitter symbol lookup error
- [ ] SequenceCoverageCalculator: symbol lookup error
- [ ] translate_bed_sequences python syntax error
- [ ] map_peptides_to_bed python syntax error
- [ ] bed_to_protein_map python syntax error
- [ ] msconvert `sudo: unknown uid 1001: who are you?`
- [ ] reactome_pathwaymatcher `mv: invalid option -- 't'`
- [ ] searchgui output size diff
- [ ] thermo_raw_file_converter `docker: invalid reference format: repository name must be lowercase.`
- [ ] meta_proteome_analyzer `Output file did not contain expected text 'A2SPK1'`
- [ ] lfq_protein_quant `In OlsonNames() : no Olson database found`
- [ ] flashlfq `Output file did not contain expected text 'EGFQVADGPLYR'`
- [ ] sixgill_build `NameError: name 'xrange' is not defined`, `AttributeError: 'pysam.libcfaidx.FastqFile' object has no attribute 'next'`
- [ ] regex1 `ModuleNotFoundError: No module named 'commands'`
- [ ] regexColumn1 `ModuleNotFoundError: No module named 'commands'`

**Comment by bgruening** [comment]
Awesome, thanks for the list. I will look at this in detail as soon as we merge your testing and have a better handle to test those.

**Comment by bernt-matthias** [comment]
Yep. I will open a separate (from the OpenMS) PR as soon as the IUC one is accepted. I think its better to stay in sync -- even if the 40 chunks may be a bit to much here .. on the other hand more tests than currently are in this repo will come with OpenMS :)

**Comment by bernt-matthias** [comment]
Weekly test are running again: we are at 48 out of 267. Planemo html output is here https://github.com/galaxyproteomics/tools-galaxyp/suites/669719012/artifacts/6016397 

Do you think its a good idea to ping the authors of the failing tools? Ping at gitter?

**Comment by bgruening** [comment]
I think we should organize a meeting soon with all GalaxyP interested people. Indeed most if the failing tests are msconvert, which needs to be run with `planemo test --docker` and the rest is not that much.

Do you think openms can be merged soon? Meeting after this and then explain the entire new infrastructure.

**Comment by bernt-matthias** [comment]
> Do you think openms can be merged soon?

Hope so. Will try to focus on the remaining points now .. just fixed (hopefully) PepNovoAdapter and IDMapper. Anyway with `profile=20.05` we are not in a rush ... but I also want to get this done. So I will see what I can do until the end of the week and then we blacklist everything else? 

Maybe we just organize the meeting before merging and discuss if anyone needs a tool that would be blacklisted.
